docs/docs-content/clusters/public-cloud/aws/eks-hybrid-nodes/create-hybrid-node-pools.md

```diff
@@ -0,0 +1,370 @@
+   64 bytes from 192.168.5.10: icmp_seq=4 ttl=63 time=30.345 ms
+
+   --- 192.168.5.10 ping statistics ---
+   4 packets transmitted, 4 packets received, 0% packet loss
+   round-trip min/avg/max/stddev = 27.359/28.875/30.345/1.091 ms
+   ```
+
+## When to Manually Repave Hybrid Node Pools
+
+<!-- This section may be removed as the workaround is not currently working. There may also be repave logic included to resolve this for 4.5.c. -->
+
+Your hybrid node pools require manual repaving in these scenarios:
+
+- After modifying the **Access Management** settings of your Amazon EKS cluster in Palette. Refer to steps 11 through 13
+  in [Import Cluster](./import-eks-cluster-enable-hybrid-mode.md#import-cluster).
+- After changing an edge host's **VPN Server IP**. Refer to step 7 in [Create Node Pool](#create-node-pool).
+
+These changes do not take effect until you repave the affected node pools.
+
+- For **Access Management** changes, repave all hybrid node pools.
+- For **VPN Server IP** changes, repave only the node pool containing the modified edge host.
+
+### Prerequisites
+
+- Your Palette account role must have the `cluster.update` permission to edit clusters. Refer to the
+  [Cluster Profile](../../../../user-management/palette-rbac/project-scope-roles-permissions.md#cluster-profile)
+  permissions for guidance.
+
+### Trigger Repave on Hybrid Node Pool
+
+Use the following steps to manually trigger a repave on a hybrid node pool.
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu**, select **Profiles**.
+
+3. On the **Profiles** page, click on your hybrid node pool cluster profile.
+
+4. You can confirm that this cluster profile is managing your hybrid node pool by viewing the **In Use Clusters** list.
+   Your hybrid node pool names are listed here.
+
+   ![Edit Hybrid Profile](/aws_eks-hybrid_create-hybrid-node-pools_in-use-clusters.webp)
+
+5. Click on the **cni-custom 0.1.0** network layer to view the **Edit Pack** page.
+
+6. In the YAML editor on the right, change the value for `manifests.byo-cni.contents.data.custom-cni` to something
+   different.
+
+   Example.
+
+   ```yaml
+   custom-cni: "dummy-repave-1"
+   ```
+
+   As mentioned in step 14 during the [Create Profile](#create-profile) steps, the specific value you enter does not
+   affect your hybrid node pool's functionality, but any change to this field will trigger the required repave.
+
+7. Click **Confirm Updates** when done, then click **Save Changes**.
+
+8. From the left **Main Menu**, select **Clusters**.
+
+9. Select your cluster to view its **Overview** tab.
+
+10. Select the **Nodes** tab.
+
+11. On the **Nodes** tab, once the profile change has been processed, an **Updates pending** banner appears. Click on
+    **Node Pool Updates** in the banner.
+
+12. On the **Pool changes summary** pop-up window, click the checkbox next to **Upcoming changes in hybridPoolName
+    configuration**. Click **Confirm** afterwards.
```

Comment:
```suggestion
    configuration**. Click **Confirm** afterward.
```
American English usage. Don't know how strict we are on that, but wanted to point it out.

Reply:
TIL